assets.sh

```diff
@@ -31,47 +34,61 @@ NCN_ARCH='x86_64'
 CN_ARCH=("x86_64")
 
 # All images must use the same, exact kernel version.
-KERNEL_VERSION='6.4.0-150600.23.17-default'
-# NOTE: The kernel-default-debuginfo package version needs to be aligned
-# to the KERNEL_VERSION. Always verify and update the correct version of
-# the kernel-default-debuginfo package when changing the KERNEL_VERSION
-# by doing a zypper search for the corresponding kernel-default-debuginfo package
-# in the SLE-Module-Basesystem update_debug repo
-# zypper --plus-repo=https://${ARTIFACTORY_USER}:${ARTIFACTORY_TOKEN}@artifactory.algol60.net/artifactory/sles-mirror/Updates/SLE-Module-Basesystem/15-SP4/x86_64/update_debug se -s kernel-default-debuginfo
-KERNEL_DEFAULT_DEBUGINFO_VERSION="${KERNEL_VERSION//-default/}.1"
+KERNEL_VERSION='6.4.0-*-default'
```

Comment:
Temporary change to test glob processing. Will be reverted before merge into release branch.